### ISSUE TITLE
Add padding for app-content-list

### ISSUE THREAD

**Issue by diemelone** [nextcloud/mail#4956]
please add 

> padding-top: 44px;

for

> .app-content-list

otherwise it is hidden behind the app-navigation-toggle

**Comment by ChristophWurst** [comment]
screenshot?

**Comment by diemelone** [comment]
![grafik](https://user-images.githubusercontent.com/46229652/115006665-72588e80-9ea9-11eb-9c48-a7ce47796025.png)

**Comment by ChristophWurst** [comment]
@GretaD could you have a look? I don't get it :see_no_evil: 

@diemelone please try to be more elaborate with your suggestions. Describe what the problem was before and how it's better now. Also with screenshots of design changes it always helps to have a before/after.

**Comment by szaimen** [comment]
@ChristophWurst I think @diemelone talks about the overlapping hamburger button that you can see in the screenshot...

**Comment by diemelone** [comment]
yes @szaimen that is what i was talking about, it happens in different apps and mail was the first one i noticed. But it happens in other apps as well.

**Comment by diemelone** [comment]
How it is:
![grafik](https://user-images.githubusercontent.com/46229652/115200748-d4f19a80-a0f4-11eb-8962-846cea1bb49c.png)

How it should be:
![grafik](https://user-images.githubusercontent.com/46229652/115200870-f94d7700-a0f4-11eb-9a94-8fefa350a7be.png)


Further examples:
Calendar 
![grafik](https://user-images.githubusercontent.com/46229652/115199978-0ddd3f80-a0f4-11eb-8a16-852a36f6183d.png)

Apps
![grafik](https://user-images.githubusercontent.com/46229652/115200130-2d746800-a0f4-11eb-86cd-a2c331d518d9.png)
There are more examples then these two but it should be enough to demonstrate the problem

**Comment by ChristophWurst** [comment]
Got it, thanks. @GretaD knows more about this as we tried to improve this in the past.

**Comment by GretaD** [comment]
@diemelone hello,
Yes, there've been couple of discussions on how to fix this overlapping and IIRC we decided not to add padding for app-content-list because it takes more space from the preview of envelops/contacts. 
But maybe we want to reconsider, @jancborchardt @skjnldsv what do you think?

**Comment by diemelone** [comment]
i am not quite sure what you mean by "preview of envelopes/contacts", but adding padding to the described html class doesnt change anything else in my installation.
Do you mean the yellow marked areas? They didn't change.
![grafik](https://user-images.githubusercontent.com/46229652/115210645-171fd980-a0ff-11eb-936d-8772b17afb53.png)

**Comment by GretaD** [comment]
> i am not quite sure what you mean by "preview of envelopes/contacts", but adding padding to the described html class doesnt change anything else in my installation.
>
No, the second column, where the envelops are shown - because of the padding added, one message is "gone" (in the end of the list)

**Comment by GretaD** [comment]
> adding the same but negative amount at the end doesn't work?

we are working on [this](https://github.com/nextcloud/mail/issues/4805#issuecomment-806750193) ticket now, which means the envelopes will be even wider to fit the tags, which means that more messages will be gone in the end of the list. Im not saying your idea is bad or wrong, I just need our design lead to agree that we can "sacrifice" 2 messages to have the hamburger menu not overlapping.

**Comment by jancborchardt** [comment]
So the original issue @diemelone described is a duplicate of https://github.com/nextcloud/server/issues/16934#issuecomment-542201187 – with the solution described in my comment. :)

So yes, basically we can add some padding at the top of the envelope list. If we like, we can additionally put something additional on the top there like:
- The "New message" button, with a small "refresh" icon on the right of it
- Or just space there – looks strange though
- The name of the current folder, although I’ve never seen that in another app and it’s just duplicated
- Something else?

**Comment by diemelone** [comment]
"New Message" would fit there quite well I think, but to keep the overall similar design (e.g. calendar app) I wouldn't change that, instead the foldername like in the photos app would be a good visual fix. 
A refresh on the other hand would also be really useful and would probably look good as well

**Comment by ChristophWurst** [comment]
> we are working on [this](https://github.com/nextcloud/mail/issues/4805#issuecomment-806750193) ticket now, which means the envelopes will be even wider to fit the tags, which means that more messages will be gone in the end of the list. Im not saying your idea is bad or wrong, I just need our design lead to agree that we can "sacrifice" 2 messages to have the hamburger menu not overlapping.

@jancborchardt did you read this?

**Comment by thstyl2000** [comment]
I also think that "delete message" "flagging" and local search would come in very very handy! (I find the frontends of sogo and afterlogic quite nice and good to work with...)

**Comment by MarkusKepert** [comment]
I have little similar issue with the toggle and i'll improved this a little bit (without much testing) and got this result as a quick fix: 
![image](https://user-images.githubusercontent.com/6367277/115364060-4bf56480-a1c3-11eb-9e99-5c60ffa9f5f5.png)

What do you think about this?

**Comment by diemelone** [comment]
for those that are looking for a quick and dirty fix I think I have got one...

Using a border-top that is 44px high, everythink works fine, I haven't lost any messages that like mentioned before and no unwanted scrollbar is created.

I used the **custom css app**. 

---

<details>
<summary>Screenshots: Padding fix</summary>
padding doesn't really do the trick as it only works for the first entry. Using margin creates an unwanted scrollbar.

before scrolling:

![grafik](https://user-images.githubusercontent.com/46229652/115373764-77308180-a1cc-11eb-8684-b92d66c553ce.png)

after scrolling:
![grafik](https://user-images.githubusercontent.com/46229652/115373858-8fa09c00-a1cc-11eb-9a38-dc8e4ccc4f38.png)

</details>


<details>
<summary>Screenshots: Border-top fix</summary>
So I added a border-top for the .app-content-list class, for demonstration purposes in red

before scrolling:
![grafik](https://user-images.githubusercontent.com/46229652/115374047-bf4fa400-a1cc-11eb-86bb-8ac0482f1406.png)

after scrolling:
![grafik](https://user-images.githubusercontent.com/46229652/115374134-d68e9180-a1cc-11eb-8cca-0a879556c4ae.png)

</details>

---

The following css takes effect on multiple apps including mail and contacts. I haven't tried many more, so I don't know all apps that are effected. For the calendar app the second css code is relevant.

```
/*mail app*/
.app-content-list {
	border-top: 44px solid #fff;
}
/*calendar app*/
.fc-media-screen {
	border-top: 44px solid #fff;
}
```

**Comment by MarkusKepert** [comment]
Thats a great fix. What i didn't see first, that this isnt a problem of mail app. It's problem of Nextcloud core CSS i think

**Comment by mat-m** [comment]
> So yes, basically we can add some padding at the top of the envelope list. If we like, we can additionally put something additional on the top there like:
> 
> * The "New message" button, with a small "refresh" icon on the right of it
> * Or just space there – looks strange though
> * The name of the current folder, although I’ve never seen that in another app and it’s just duplicated
> * Something else?

* Refresh button would be useful (And would be perfect if it also includes reload to complete the list of messages after having deleted some)
* Select all button, maybe ?

**Comment by jancborchardt** [comment]
Currently there is a fix which makes it look halfway done, with there now being an empty "header bar" which is only there for the menu toggle:
![image](https://user-images.githubusercontent.com/925062/154269860-805f9c17-a574-4f49-b8a5-c93429e0811a.png)

The proper solution would be to move the "New message" and "Refresh" buttons in from the navigation to that area as well, as specified in: https://github.com/nextcloud/mail/issues/4956#issuecomment-822330614
(Same for Contacts)

**Comment by reos-rcrozier** [comment]
Another solution would be something more like this:
![image](https://user-images.githubusercontent.com/91322120/157686828-5da797b3-58f4-4e4f-868d-d21277493f75.png)

Where you always loose a thin little bit of space to the left, and the button actually stays on the menu side and therefore never interferes with the content.
